activerecord/lib/active_record/attributes.rb

```diff
@@ -112,6 +115,15 @@ module ClassMethods
       #       my_float_range: 1.0..3.5
       #     }
       #
+      # Passing options to the type constructor
+      #
+      #   # app/models/my_model.rb
+      #   class MyModel < ActiveRecord::Base
+      #     attribute :small_int, :integer, limit: 2
+      #   end
+      #
+      #   MyModel.create(small_int: 65537)
+      #   # => Error: 65537 is out of range for the limit of two bytes
```

Comment:
I think there should be a line break here. Can you add it?

Reply:
Added!